src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -7682,9 +7682,11 @@ private static void CombineExtensionMethodArguments(BoundExpression receiver, An
             {
                 Debug.Assert(lookupResult.Symbols.Any());
                 var members = ArrayBuilder<Symbol>.GetInstance();
-                bool wasError;
-                Symbol symbol = GetSymbolOrMethodOrPropertyGroup(lookupResult, node, rightName, arity, members, diagnostics, out wasError, qualifierOpt: null);
+
+#if DEBUG
+                Symbol symbol = GetSymbolOrMethodOrPropertyGroup(lookupResult, node, rightName, arity, members, diagnostics, out _, qualifierOpt: null);
```

Comment:
This actually have side effect (it adds symbols to `members` for example) :/

Reply:
I'm closing this PR, but in case someone sees a potential improvement to reduce allocations in this path, let me know.

cc @jcouv @cston @sharwell